apps/src/lab2/views/components/Instructions/instructions.module.scss

```diff
@@ -54,14 +54,24 @@
 
 .bubble {
   position: relative;
-  padding: 12px;
   border-radius: 4px;
   animation: slidein 0.5s;
   background-color: var(--background-neutral-secondary);
   color: var(--text-neutral-primary);
   display: flex;
   flex-direction: column;
-  min-height: 0; 
+  min-height: 0;
+
+  /* If there is scrolling content, it will provide horizontal padding.
+     Doing this allows the scrollbar to use that padding area. */
+  &:has(.scrollingContent) {
```

Comment:
It appears that this is causing a regression for the 'Validate' button in `pythonlab`.
<img width="921" height="450" alt="Screenshot 2025-09-19 at 3 45 35 PM" src="https://github.com/user-attachments/assets/7038d9a4-8556-4a45-bc1c-b9783e7bc11d" />

Note that if `pythonlab` is using the resource panel, the padding is correct! But I am unsure when `pythonlab` will migrate to the resource panel.
<img width="938" height="351" alt="Screenshot 2025-09-19 at 3 47 08 PM" src="https://github.com/user-attachments/assets/28748f57-3062-4cf8-9af7-d362f9c4fd01" />